dev/com.ibm.ws.security.fat.common.jwt/.classpath

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
 	<classpathentry kind="src" path="src"/>
+	<classpathentry kind="src" path="fat/src"/>
 	<classpathentry kind="src" path="test-applications/jwtbuilder/src"/>
 	<classpathentry kind="con" path="aQute.bnd.classpath.container"/>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
+	<classpathentry kind="lib" path="/com.ibm.ws.org.jose4j/build/libs/com.ibm.ws.org.jose4j.jar"/>
```

Comment:
This shouldn't be needed - anything needed to build should be in the project's bnd.bnd file. A local Eclipse instance might've been satisfied, but it should be understanding and getting everything it needs from bnd.bnd.